src/app.py

```diff
@@ -291,11 +338,13 @@ def upload_csv():
         )
 
     # make sure these columns aren't empty
-    if data[["uuid", "name"]].isna().sum().sum() > 0:
-        logger.error("uuid or name column values are empty!")
+    if data[["uuid", "name", "sealed_state"]].isna().sum().sum() > 0:
```

Comment:
Why double sum?

Reply:
This is to make sure any of the given columns are not null. the first sum() will give us the sum per column. And applying sum again to that gives us a single value.